diff --git a/scripts/restore.sh b/scripts/restore.sh
index 1a5e3f9..46e4778 100755
--- a/scripts/restore.sh
+++ b/scripts/restore.sh
@@ -378,7 +378,6 @@ main() {
 		restore_grouped_sessions  # also restores active and alt windows for grouped sessions
 		restore_active_and_alternate_windows
 		restore_active_and_alternate_sessions
-		cleanup_restored_pane_contents
 		execute_hook "post-restore-all"
 		stop_spinner
 		display_message "Tmux restore complete!"
